src/Init/Util.lean

```diff
@@ -85,3 +85,22 @@ private def outOfBounds [Inhabited α] : α :=
 
 macro:max x:term noWs "[" i:term "]" noWs "?" : term => `(getElem? $x $i)
 macro:max x:term noWs "[" i:term "]" noWs "!" : term => `(getElem! $x $i)
+
+/--
+  Marks given value and its object graph closure as multi-threaded if currently
+  marked single-threaded. This will make reference counter updates atomic and
+  thus more costly. It can still be useful to do eagerly when the value will be
+  shared between threads later anyway and there is available time budget to mark
+  it now. -/
+@[extern "lean_runtime_mark_multi_threaded"]
+def Runtime.markMultiThreaded (a : α) : α := a
+
+/--
+  Marks given value and its object graph closure as multi-threaded if currently
+  marked single-threaded. This will remove reference counter updates but prevent
```

Comment:
Is this first sentence deliberately identical to the one in the other docstring?

Reply:
Wow, thanks, I must have overwritten it in a later revision